cmd/gpumetric/main.go

```diff
@@ -0,0 +1,145 @@
+// Copyright 2020, OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"errors"
+	"log"
+	"net/http"
+	"time"
+
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/client_golang/prometheus/promhttp"
+)
+
+// GPU metrics declarations
+var (
+	gpuTemp = prometheus.NewGauge(prometheus.GaugeOpts{
```

Comment:
Why not using otel golang?

Reply:
and push things using otlp exporter?

Reply:
@bogdandrutu sorry for delayed response. I was taking a vacation. So do you mean instrumenting with OTel Go SDK and send data with OTLP exporter to the Collector instead of Prometheus + Prometheus receiver?